api/proto/teleport/notifications/v1/notifications_service.proto

```diff
@@ -109,3 +114,21 @@ message UpsertUserLastSeenNotificationRequest {
   // user_notification_state is the updated user last seen notification item.
   UserLastSeenNotification user_last_seen_notification = 2;
 }
+
+// ListAllUserCreatedNotificationsForUserRequest is the request for listing all user-created notifications for a specific user.
+message ListAllUserCreatedNotificationsForUserRequest {
+  // page_size is the size of the page to return.
+  int32 page_size = 1;
+  // page_token is the next_page_token value returned from a previous ListUserNotifications request, if any.
+  string page_token = 2;
+  // username is the username of the user the notifications being listed are for.
+  string username = 3;
+}
+
+// ListAllUserCreatedGlobalNotificationsRequest is the request for listing all user-created global notifications..
```

Comment:
What is a user-created global notification?

Reply:
Any global notification which was created by a user using `tctl notifications create` (as opposed to global notifications generated by Teleport, eg. access requests). The main use of the `tctl notifications` interface is to create and manage custom notifications, so I didn't think it would be necessary to include _all_ global notifications as that would clutter the list with dozens of irrelevant notifications.

Reply:
If there are dozens of global notifications that would clutter this list, wouldn't that also be cluttering every user's notification list in the UI?

`tctl` is an admin tool more than it is a user tool. I think its acceptable (and preferred) for an admin tool to show you everything rather than hide stuff from you.

Reply:
> wouldn't that also be cluttering every user's notification list in the UI?

No, since for regular users in the UI, only global notifications which match the user would be shown to them, ie. for access request notifications, a user in the UI will only see the notifications relevant to them (that their request was approved/denied or that a request is pending their review)